worlds/pokemon_rb/rules.py

```diff
@@ -3,7 +3,7 @@
 from . import logic
 
 
-def set_rules(multiworld, player):
+def set_rules(multiworld, world, player):
```

Comment:
if you have the world, you don't need the player, it's in it

Reply:
I don't need to set player to world.player on a separate line of code when I can do this.